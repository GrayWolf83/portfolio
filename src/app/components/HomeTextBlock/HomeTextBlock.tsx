import React from 'react'
import { Box, Typography } from '@mui/material'

interface IHomeTextBlock {}

const HomeTextBlock: React.FC<IHomeTextBlock> = () => {
	return (
		<div data-testid='HomeTextBlock'>
			<Box
				sx={{
					mt: '20px',
					display: 'flex',
					justifyContent: 'center',
					alignItems: 'center',
				}}>
				<Box sx={{ ml: '20px' }}>
					<Typography color='primary' variant='h4'>
						Сергей Бернякович
					</Typography>
					<Typography color='primary' variant='h5'>
						Frontend разработчик (React)
					</Typography>
				</Box>
			</Box>
			<Typography
				variant='body1'
				sx={{ my: '20px', textAlign: 'justify', fontWeight: 'bold' }}
				color='primary'>
				Данный сервис преобразовывает информацию о пользователях GitHub
				в удобное портфолио и может использоваться рекрутерами для
				просмотра аккаунтов и оценки уровня кандидатов.
			</Typography>
		</div>
	)
}

export default HomeTextBlock
