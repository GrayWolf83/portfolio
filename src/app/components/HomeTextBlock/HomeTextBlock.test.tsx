import React from 'react'
import { render, screen } from '@testing-library/react'
import { BrowserRouter } from 'react-router-dom'
import HomeTextBlock from './HomeTextBlock'

describe('HomeTextBlock component', () => {
	test('HomeTextBlock render', () => {
		render(
			<BrowserRouter>
				<HomeTextBlock />
			</BrowserRouter>,
		)

		const homeTextBlock = screen.getByText(/Frontend/)
		expect(homeTextBlock).toBeInTheDocument()
	})

	test('HomeTextBlock snapshot', () => {
		expect(
			render(
				<BrowserRouter>
					<HomeTextBlock />
				</BrowserRouter>,
			),
		).toMatchSnapshot()
	})
})
