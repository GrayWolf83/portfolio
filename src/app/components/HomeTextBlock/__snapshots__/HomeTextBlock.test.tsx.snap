// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`HomeTextBlock component HomeTextBlock snapshot 1`] = `
Object {
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <div
        data-testid="HomeTextBlock"
      >
        <div
          class="MuiBox-root css-cph0fm"
        >
          <div
            class="MuiBox-root css-s8vbhp"
          >
            <h4
              class="MuiTypography-root MuiTypography-h4 css-11kybd2-MuiTypography-root"
            >
              Сергей Бернякович
            </h4>
            <h5
              class="MuiTypography-root MuiTypography-h5 css-1invrju-MuiTypography-root"
            >
              Frontend разработчик (React)
            </h5>
          </div>
        </div>
        <p
          class="MuiTypography-root MuiTypography-body1 css-764ec4-MuiTypography-root"
        >
          Данный сервис преобразовывает информацию о пользователях GitHub в удобное портфолио и может использоваться рекрутерами для просмотра аккаунтов и оценки уровня кандидатов.
        </p>
      </div>
    </div>
  </body>,
  "container": <div>
    <div
      data-testid="HomeTextBlock"
    >
      <div
        class="MuiBox-root css-cph0fm"
      >
        <div
          class="MuiBox-root css-s8vbhp"
        >
          <h4
            class="MuiTypography-root MuiTypography-h4 css-11kybd2-MuiTypography-root"
          >
            Сергей Бернякович
          </h4>
          <h5
            class="MuiTypography-root MuiTypography-h5 css-1invrju-MuiTypography-root"
          >
            Frontend разработчик (React)
          </h5>
        </div>
      </div>
      <p
        class="MuiTypography-root MuiTypography-body1 css-764ec4-MuiTypography-root"
      >
        Данный сервис преобразовывает информацию о пользователях GitHub в удобное портфолио и может использоваться рекрутерами для просмотра аккаунтов и оценки уровня кандидатов.
      </p>
    </div>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
